lightly_studio_view/src/lib/components/Header/Header.test.ts

```diff
@@ -10,6 +10,18 @@ import type { CollectionView } from '$lib/api/lightly_studio_local';
 
 import { useRootCollectionOptions } from '$lib/hooks/useRootCollection/useRootCollection';
 
+vi.mock('$lib/hooks/useHasEmbeddings/useHasEmbeddings', async () => {
```

Comment:
There was an issue in tests, they tried to hit an endpoint. This was suggested by AI. Is this the proper way?